### ISSUE TITLE
Add Alfredo Metatopia

### ISSUE THREAD

**Issue by DrAlta** [opengaming/osgameclones#986]
name: Alfredo Metatopia
originals:
- Prom Week
- Siboot
- The Sims
- Dungeon Keeper
type: clone
repo: 'https://github.com/users/DrAlta/projects/1'
url: 'http://yiffiemon.wikia.com/'
development: active
status: unplayable
multiplayer:
- Online
content: open
lang:
- Python
license:
- AGPL3
info: >-
  A serious game using psychosocial models from academia. Aims to take The Sims
  in the opposite direction of The Sims 4 with Massively  Single Player 
  approach to creating the characters you want then letting them run their own lives
  instead of micro managing their lives for them.
updated: '2019-11-14'

**Comment by cxong** [comment]
There's no code for this project yet. Can reconsider this once the project takes shape

**Comment by DrAlta** [comment]
In case anyone comes across this. I decided to split it up into a series of progressively more complex games. I also moved the website over to ourproject.org and the code to GitGud https://gitgud.io/QuiteUnusualDev since I want to make a Relationship and Sex Education game to teach Imago Relationship theory. Mostly, I'm waiting for Godot 4 to be released.  

I'll resubmit the individual games as I work on them.  

Thanks for your time.
